src/Main.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Daniel Church
+ * Daniel Church foo
```

Comment:
```suggestion
 * Daniel Church Foo
```

Foo really deserves to be capitalized here, especially since I believe this is just an extension of your name now